modules/ztp-pre-staging-creating-rhcos-live-iso.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-pre-staging-tool.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-create-custom-rhcos-live-iso_{context}"]
+= (Optional) Creating a custom RHCOS live ISO
+
+It is recommended to permit logging into the server through SSH before partitioning.
```

Comment:
I'll say something like: "The partitioning of the disk (section) and downloading of the images (section) are executed in the server that is going to be provisioned, therefore it is recommended..." . I mean I am missing the why. We can also compare with executing the commands through the console, but it is tedious as mentioned somewhere else.

Since it is (optional) we should explain what will happen if we do not create the custom ISO. Basically you will connect through a virtual console or console. As said this is the recommended way and all the examples in the doc are taking into account you're using the custom iso.